configure.ac

```
@@ -128,6 +128,12 @@ AC_ARG_ENABLE([wallet],
   [enable_wallet=$enableval],
   [enable_wallet=yes])
 
+AC_ARG_WITH([sqlite],
+  [AS_HELP_STRING([--without-sqlite],
+  [disable sqlite wallet support (default is enabled if wallet is enabled and sqlite is found)])],
```

5c19bd0f4b893a91c02e652ac8f45ef0ccdeca1c

Mind avoiding negative form in help string:
```suggestion
  [AS_HELP_STRING([--with-sqlite=yes|no|auto],
  [enable sqlite wallet support (default: auto, i.e., enabled if wallet is enabled and sqlite is found)])],
```
?

---

Done